src/build.py

```diff
@@ -161,8 +162,11 @@ def NodePlatformName():
 
 # Known failures.
 IT_IS_KNOWN = 'known_gcc_test_failures.txt'
-LLVM_KNOWN_TORTURE_FAILURES = [os.path.join(LLVM_SRC_DIR, 'lib', 'Target',
-                                            'WebAssembly', IT_IS_KNOWN)]
+LLVM_KNOWN_TORTURE_FAILURES_S = [os.path.join(LLVM_SRC_DIR, 'lib', 'Target',
+                                              'WebAssembly', IT_IS_KNOWN)]
+# TODO(sbc): Move this failures file to the llvm repo
```

Comment:
*Should* this get moved to llvm? I'd assume upstream llvm wouldn't care about the gcc torture tests, so this seems like it could just live in the waterfall repo.